crates/aptos/e2e/cases/stake.py

```diff
@@ -0,0 +1,48 @@
+# Copyright © Aptos Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+
+from common import TestError
+from test_helpers import RunHelper
+from test_results import test_case
+
+
+@test_case
+def test_stake_initialize_stake_owner(run_helper: RunHelper, test_name=None):
+    # run the initialize-stake-owner command
+    response = run_helper.run_command(
+        test_name,
+        [
+            "aptos",
+            "stake",
+            "initialize-stake-owner",
+            "--initial-stake-amount",
+            "100000000000",
+            "--assume-yes",
+        ],
+    )
+
+    result = json.loads(response.stdout)["Result"]
+    if result.get("success") != True:
+        raise TestError("Did not initialize stake owner successfully")
```

Comment:
We should probably do more than just check that it's a success, looking for that it has fields that we want in the output (maybe not exactly what they say)

Reply:
Resolved on #9284